assets/scss/components/_forms.scss

```diff
@@ -30,6 +32,21 @@ input:focus + .icon--field {
 	font-weight: normal;
 }
 
+.label--required:after {
```

Comment:
Why aren't we replacing SASS vars with custom properties in here?

Reply:
rather than replacing all Sass vars, I remapped them in sasstools v4:
https://github.com/meetup/swarm-sasstools/compare/master...release/4.0.0

All existing Sass vars now map to custom properties, which allows us to launch faster and clean up in steps afterward.

Reply:
I completely forgot about that 🤦‍♂️